docker/hooks/build

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# remove trailing slash if exists
+DOCKER_REPO=${DOCKER_REPO%/}
+
+
+echo "||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||"
+date
+echo building image \"$IMAGE_NAME\" in repository \"$DOCKER_REPO\" using dockerfile \"$DOCKERFILE_PATH\"
+echo image was triggered by automated build for tag \"$DOCKER_TAG\"
+echo " based on branch \"$SOURCE_BRANCH\"  and commit \"$SOURCE_COMMIT\""
+echo " commit message was \"$COMMIT_MSG\""
+echo && echo && echo
+echo "||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||"
+
+
+./dev-build-images.sh "$DOCKER_REPO/"
+status=$?
+
+if [[ $status -eq 0 ]]; then
+  echo "#### pushing images ####"
+  ./dev-build-images.sh "$DOCKER_REPO/"
```

Comment:
Isn't this the same commad as above? Did you maybe meant to write `dev-push-images.sh` instead? (sorry if i don't get it…)

Reply:
you got it completely right, fixed ;-)